crud/src/main/java/com/redhat/lightblue/mediator/Mediator.java

```diff
@@ -733,6 +802,7 @@ public BulkResponse bulkRequest(BulkRequest requests) {
             Error.pop();
             return response;
         } finally {
```

Comment:
Should probably be adding these to the response I guess too? Now I'm wondering why that wasn't being done already.